fingerprint2.js

```diff
@@ -899,6 +871,31 @@
         return null
       }
     },
+    getWebGlShaderPrecisionFormat: function () {
+      try {
+        var gl = this.getWebglCanvas()
+        if (!gl) { return null }
+        var resultArray_ = []
+        for (const numType of ['FLOAT', 'INT']) {
```

Comment:
Don't use `for ... of` loops, it's not supported by all browsers. My suggestion is to keep the unfolded, hard-coded list of combinations, i.e.

```
[
['FLOAT', 'VERTEX', 'HIGH', 'precision'],
['FLOAT', 'VERTEX', 'HIGH', 'rangeMin'],
...
['FRAGMENT', 'LOW', 'rangeMax']
]
```

Or use `.forEach`, which is polyfilled for browsers that don't support it.